### DISCUSSION TITLE
"Built with Solito" page in docs

### DISCUSSION THREAD

**Discussion by danger9224** [nandorojo/solito#483] [Announcements]
I think it'd be really nice to have a "Built with Solito" page on the docs, kinda similar to [Flutter's "Showcases" page](https://flutter.dev/showcase), or how [Next.js](https://nextjs.org/showcase) and [React-Native's](https://reactnative.dev/showcase) websites display some highlighted apps that the community built with the respective packages. That could possibly help people convince that using Solito isn't as odd/rare as it might seem like, possibly convincing more people to try it?

**Comment by nandorojo** [comment]
i like the idea!
